doc/user/content/cloud/account-limits.md

```diff
@@ -9,21 +9,37 @@ menu:
 
 {{< cloud-notice >}}
 
+Materialize Cloud offers two deployment sizes for free trials, with additional processing and RAM available for a premium upgrade. We use AWS as the cloud service provider and the us-east-1 datacenter.
+
+Trials last for 30 days or until you max out the available resources for free deployments, whichever comes first. Accounts and deployments will be deleted 14 days after your trial expires unless you have upgraded to a premium version.
+
+## Materialize Cloud specifications
+
+Deployment Size | XS | S | M | L | XL
+----------------|----|---|---|---|---
+**Level**  | free  | free  | premium  | premium  |  premium
+**No. deployments** | 2  | 2  | ?? | ??  |  ??
```

Comment:
@msudberg 

We don't currently have a number of deployments mapping to the sizes at this time (the sizing refers to the per deployment compute and memory resourcing for now) - per my earlier comment, users who want more than two deployments will have to reach out to the Materialize team to place an 'order'.